src/GafferSceneUI/AttributeInspector.cpp

```diff
@@ -302,6 +302,13 @@ Gaffer::ValuePlugPtr AttributeInspector::source( const GafferScene::SceneAlgo::H
 		}
 	}
 
+	else
+	{
+		editWarning = boost::str(
+			boost::format( "Edits cannot be made to a %s node" ) % history->scene->node()->typeName()
+		);
```

Comment:
Perhaps the node name is more useful to the user than the node type? The latter can be discovered if you know the former, but not vice versa.

Reply:
With the reworking in 777c8c46dcada80695dacde13270b61adc29e620, I didn't feel like attributing the lack of a source to any particular node was quite right, so I left it out. There could be an argument for using the topmost node's name as I was before though.